fetch.js

```diff
@@ -189,6 +189,28 @@ function CheckSwaggerApiGroupList(playFabUrl, tocObj) {
     GetFileFromUrl(swaggerApiListUrl, options);
 }
 
+function CheckAzureApiSpecGroupList(playFabUrl, tocObj) {
+    var azureApiSpecListUrl = playFabUrl + "apispec/azure/";
+    var options = {
+        description: "azureApiSpecList",
+        expectJson: true,
+        jsonTabSpaces: 0
+        // TODO: Add a function to compare tocObj with the json result
+    };
+    GetFileFromUrl(azureApiSpecListUrl, options);
+}
+
+function CheckAzureApiSpecGroupList(playFabUrl, tocObj) {
+    var azureSwaggerApiListUrl = playFabUrl + "swagger/azure/";
```

Comment:
Just an engineering optimization opportunity, Feel free to ignore since it works as is: I feel like we could simplify these two functions with either parameters for the url/options and integrate with some of the other similar functions in this file. It would make the file a bit cleaner code wise, but I see that this works as is, so it may be dangerous to attempt to modify/simplify the legacy behavior.